python/lsst/afw/image/image/image.cc

```diff
@@ -150,6 +169,8 @@ static PyImage<PixelT> declareImage(py::module &mod, const std::string &suffix)
     cls.def("scaledMultiplies", &Image<PixelT>::scaledMultiplies);
     cls.def("scaledDivides", &Image<PixelT>::scaledDivides);
 
+    cls.def("subset", &Image<PixelT>::subset, "bbox"_a, "origin"_a=PARENT);
```

Comment:
Maybe; it's a philosophical question about the degree to which we want to optimize our Python and C++ APIs individually vs. trying to make their interfaces somewhat similar to minimize the cognitive shift when shifting from one codebase to the other.  At least when I was writing this, I leaned towards the latter, but my feelings aren't super strong in this case.

Reply:
Yeah, I guess since I've worked with the C++ code less I'm looking at it from a python user interface standpoint. But I also don't have strong feelings about this.